lib/health_check/middleware_health_check.rb

```diff
@@ -6,32 +6,30 @@ def initialize(app)
     end
 
     def call(env)
-      uri = env['PATH_INFO']
-      if uri =~ /^\/?#{HealthCheck.uri}\/([-_0-9a-zA-Z]*)middleware_?([-_0-9a-zA-Z]*)(\.(\w*))?/
-        checks = $1 + ($1 != '' && $2 != '' ? '_' : '') + $2
-        checks = 'standard' if checks == ''
-        response_type = $4
+      if env['PATH_INFO'] =~ /^\/?#{HealthCheck.uri}\/?([-_0-9a-zA-Z]*)(\.(\w*))?/
```

Comment:
How about changing a bit this pattern?
From `/^\/?#{HealthCheck.uri}\/?([-_0-9a-zA-Z]*)(\.(\w*))?/` 
->     `/^\/?#{HealthCheck.uri}(\/([-_0-9a-zA-Z]*))?(\.(\w*))?$/`?
This causes that if i.e. `HealthCheck.uri = 'health'`, such call will be caught `/health/standard.json` and `/health_check/standard.json` will not.